includes/admin/post-types/meta-boxes/class.llms.meta.box.voucher.php

```diff
@@ -45,7 +45,8 @@ public function get_fields() {
 				'fields' => array(
```

Comment:
add version changelog tags to updated methods and to file header docblocks

Reply:
fixed in https://github.com/gocodebox/lifterlms/pull/832/commits/bc7ca7152db7783d7ca1b91c4471ed8389acb90a